.envrc

```diff
@@ -3,3 +3,6 @@ eval "$(dev-env/bin/dade assist)"
 
 # Load private overrides
 [[ -f .envrc.private ]] && source_env .envrc.private
+
+# install pre-commit hook (opt-out by setting `DADE_NO_PRE_COMMIT`)
+[ -v DADE_NO_PRE_COMMIT ] || pre-commit install
```

Comment:
I get an error here

```sh
./.envrc: line 8: [: -v: unary operator expected
```

Is this zsh-only syntax?

Reply:
I'm on zsh; I don't get an error but setting that var doesn't seem to disable anything.

Reply:
Perhaps instead

```diff
-[ -v DADE_NO_PRE_COMMIT ] || pre-commit install
+test "x$DADE_NO_PRE_COMMIT" = x && pre-commit install
```

Reply:
@S11001001 please see https://github.com/digital-asset/daml/pull/15827

Reply:
Oh I do get the same error message as Stephen, I just hadn't noticed it. I'm also on macOS so also using antediluvian Bash when outside Nix.